<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Food tracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_scan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_eaten_today">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>691</width>
      <height>411</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>85</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cals</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prot</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Carbs (sug)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fat (sat)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Salt</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>528</width>
      <height>359</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="Finland"/>
    </property>
    <property name="gridVisible">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
